tests/ttnn/unit_tests/operations/eltwise/test_eltwise_sqrt.py

```diff
@@ -23,7 +23,7 @@ def run_eltwise_sqrt_tests(
     device,
 ):
     torch.manual_seed(data_seed)
-    x = torch.Tensor(size=input_shape[0]).uniform_(-100, 100).to(torch.bfloat16)
+    x = torch.Tensor(size=input_shape[0]).uniform_(1, 100).to(torch.bfloat16)  # Range will be updated after #14077.
 
```

Comment:
Why after 14077?

Reply:
We want pow to return NaN, but in TTNN, it currently returns inf. This behavior is being addressed in #14077. In the meantime, I have updated the range accordingly.